ex1.html

```diff
@@ -18,53 +20,57 @@
 					<br><br>
 					<input type="password" placeholder="Password" name="pass_box" class="loginboxes">
 					<br><br><br><br><br><br><br>
-						<a href="ex2.html" class="loginbutton">Login</a>
+					<a href="ex2.html" class="loginbutton">Login</a>
 				</form> 
 			</div>
 		</div>
 	</section>
 
 	<section>
-		<div>
-			<p><img src="new-feedback.jpg" style="float:left" >
+		<div class="div1">
```

Comment:
I figured out today, from Valy's example, that i can use a class for different divs, or nested classes.